platform/iphone/gl_view.mm

```diff
@@ -284,20 +284,28 @@ - (id)initGLES {
 			kEAGLColorFormatRGBA8,
 			kEAGLDrawablePropertyColorFormat,
 			nil];
-
-	// Create a context based on the gl driver from project settings
+	bool fallback_gl2 = false;
+	// Create a GL ES 3 context based on the gl driver from project settings
 	if (GLOBAL_GET("rendering/quality/driver/driver_name") == "GLES3") {
 		context = [[EAGLContext alloc] initWithAPI:kEAGLRenderingAPIOpenGLES3];
 		NSLog(@"Setting up an OpenGL ES 3 context. Based on Project Settings \"rendering/quality/driver/driver_name\"");
-	} else if (GLOBAL_GET("rendering/quality/driver/driver_name") == "GLES2") {
-		context = [[EAGLContext alloc] initWithAPI:kEAGLRenderingAPIOpenGLES2];
-		gles3_available = false;
-		NSLog(@"Setting up an OpenGL ES 2 context. Based on Project Settings \"rendering/quality/driver/driver_name\"");
+		if (!context && GLOBAL_GET("rendering/quality/driver/fallback_to_gles2")) {
+			gles3_available = false;
+			fallback_gl2 = true;
+			NSLog(@"Failed to created OpenGL ES 3 context. Falling back to OpenGL ES 2.");
```

Comment:
Typo here, "created".

Reply:
Since you have to amend, might be worth using "2.0" and "3.0" in those logs to be explicit, not just "2" and "3" (as we're not using OpenGL ES 3.1 or 3.2).